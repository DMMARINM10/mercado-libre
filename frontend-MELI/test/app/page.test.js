import Home from '@/app/page';
import { render } from '@testing-library/react';

describe('Home Page component', () => {
  it('should match the snapshot', () => {
    const { asFragment } = render(<Home />);
    expect(asFragment()).toMatchSnapshot();
  });
});