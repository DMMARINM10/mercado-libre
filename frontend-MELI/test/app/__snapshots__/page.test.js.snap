// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page component should match the snapshot 1`] = `<DocumentFragment />`;
